apps/server/README.md

```diff
@@ -87,25 +95,26 @@ The .env file you pass to this command can follow the formatting of the template
-| MONGO_HOST         | No                             | Number       | `27017`     | Port of MongoDB                                                                                                                             |
-| MONGO_USER         | No                             | String       | `admin`     | User name for MongoDB Connection                                                                                                            |
-|                    |                                |              |             |                                                                                                                                             |
-| VAULT_ENABLED      | No                             | Boolean      | `false`     | Set to true to enable reading secret values from Vault. See [Vault for Secret Storage](#vault-for-secret-storage).                          |
-| VAULT_ROLE         | When `VAULT_ENABLED` is `true` | String       | -           | Role to use for Vault connection, needs permission to read from `VAULT_SECRETS_PATH`                                                        |
-| VAULT_SECRETS_PATH | When `VAULT_ENABLED` is `true` | String       | -           | Path to location in Vault that holds Lectern relevant secrets                                                                               |
-| VAULT_TOKEN        | When `VAULT_ENABLED` is `true` | String       | -           | Access Token to read from Vault using specified `VAULT_ROLE`                                                                                |
+| Variable Name        | Required                       | Type         | Default     | Description                                                                                                                                             |
+| -------------------- | ------------------------------ | ------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| OPENAPI_PATH         | No                             | String       | `/api-docs` | Path to Swagger UI with API documentation.                                                                                                              |
+| PORT                 | No                             | Number       | `3000`      | Port Lectern Server API will listen to.                                                                                                                 |
+| CORS_ALLOWED_ORIGINS | No                             | String[]     | -           | List of domains that will be allowed by CORS. Multiple domains can be listed, separated by commas. Example: `http://localhost:5173,https://example.com` |
```

Comment:
ah, looks like this is the answer to my earlier question. perhaps examples would be useful in the `.env.example`? (unless the intent was to create a `.env.template` instead)
```suggestion
| CORS_ALLOWED_ORIGINS | No                             | String[]     | -           | List of domains that will be allowed by CORS. Multiple domains can be listed, separated by commas. No spaces. Example: `http://localhost:5173,https://example.com` |
```

Reply:
I'll add the trim thing instead.